pumpkin-world/src/chunk/mod.rs

```diff
@@ -91,28 +112,30 @@ pub struct ChunkHeightmaps {
     world_surface: LongArray,
 }
 
-#[derive(Deserialize, Debug)]
-#[expect(dead_code)]
+#[derive(Serialize, Deserialize, Debug)]
 struct ChunkSection {
     #[serde(rename = "Y")]
-    y: i32,
+    y: i8,
     block_states: Option<ChunkSectionBlockStates>,
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 #[serde(rename_all = "PascalCase")]
 struct ChunkNbt {
-    #[expect(dead_code)]
-    data_version: usize,
-
+    data_version: i32,
+    status: ChunkStatus,
+    // #[serde(rename = "xPos")]
```

Comment:
What were these? can they be removed?

Reply:
All things in there are NBT data stored in the chunk